docker/build_artifacts/sagemaker/tensorflow-serving.js

```diff
@@ -77,10 +77,12 @@ function tfs_json_request(r, json) {
             body = body.replace("\\'instances\\'", "'instances'")
         }
 
-        var content_types = accept.trim().replace(" ", "").split(",")
-        if (content_types.includes('application/jsonlines') || content_types.includes('application/jsons' == accept)) {
-            body = body.replace(/\n/g, '')
-            r.headersOut['Content-Type'] = content_types[0]
+        if (accept != undefined) {
+            var content_types = accept.trim().replace(" ", "").split(",")
```

Comment:
What is expected type of accept variable and does that type have a trim method?
It is also not clear if replace is safe - did we actually meant to do both left and right trim bot not touch any blank space in between - if not, then please explain the scenario this is trying to fix and why we think the fix is safe.
Thanks!